<launch>
  <node name="autonomy_simulator" pkg="autonomy_simulator" type="virtual_dc_motor" output="log" respawn="true"></node>
  <!-- You can add your nodes here to launch them! :) -->
</launch>
